include/openssl/ssl.h

```diff
@@ -253,15 +251,15 @@ typedef int (*tls_session_secret_cb_fn) (SSL *s, void *secret,
 /* Typedefs for handling custom extensions */
 
 typedef int (*custom_ext_add_cb) (SSL *s, unsigned int ext_type,
```

Comment:
and remove the spaces between the function name and the param lists here, and below, as well.

Reply:
Too much too fix ;)